tests/test_toggle.py

```diff
@@ -336,6 +358,38 @@ def test_dynamic_to_json(self):
         self.assertEqual(json.loads(json.dumps(data)), golden)
 
 
+class ToggleLoadTC(unittest.TestCase):
+
+    def test_load(self):
+        fixture = '''[{"fixed": {"show_axis": false}},
+{"dynamic": {"apps": {"euler1d": {"use_sub": false}}}}]'''
+        tg = modmesh.toggle.load(
+            fixture,
+            toggle_instance=modmesh.Toggle.instance.clone())
+        self.assertEqual(tg.apps.euler1d.use_sub, False)
+
+        fixture = '''[{"fixed": {"show_axis": false}},
+{"dynamic": {"apps": {"euler1d": {"use_sub": true}}}}]'''
+        tg = modmesh.toggle.load(
+            fixture,
+            toggle_instance=modmesh.Toggle.instance.clone())
+        self.assertEqual(tg.apps.euler1d.use_sub, True)
+
+    @unittest.skip("the lifecycle issue may cause segfault")
```

Comment:
I did something wrong ... This test function is to remind us that I need to fix the mistake I made.